test_files/es6/optional.js

```diff
@@ -0,0 +1,2 @@
+function OptionalArgument(/** number */ x, /** string= */ y) {
```

Comment:
I just meant the test case, this is a 100% pure small batch hand made nitpick :-)